README.adoc

```diff
@@ -183,7 +146,7 @@ Use the `-t` flag to give the image an optional name. In this case, `ol-runtime`
 The first build usually takes much longer to complete than subsequent builds because Docker needs to
 download all dependencies that your image requires, including the parent image.
 
-If your build runs successfully, an output appears that is similar to the following example:
+If your build runs successfully, you will see an output similar to the following:
```

Comment:
...you **see** an output..."

Reply:
I might be wrong, but doesnt this fall under the 1st type conditional sentences? Meaning, the outcome is likely but not guaranteed. In this case, the build will likely succeed, but its not for certain since the user might have done something wrong. I took it from here https://www.grammarly.com/blog/conditional-sentences/.

Reply:
How about using "you'll" instead of "you will"? That way, we keep the conditional rule, but we make the "will" less obtrusive, plus we make it more conversational because I tend to think of contractions as more conversational. It's a win-win all around, right? ;)